src/client/components/app/Transactions/AddCreditPositionTx.tsx

```diff
@@ -117,7 +118,7 @@ export const AddCreditPositionTx: FC<AddCreditPositionProps> = (props) => {
         })
       );
     }
-    if (selectedTokenAddress === '' && selectedSellToken) {
+    if ((selectedTokenAddress === '' || !selectedTokenAddress) && selectedSellToken) {
```

Comment:
these are the same thing. can u just make it `!selectedTokenAddress` plz

Reply:
No, these are not the same. 

There are instances where `selectedTokenAddress` is `undefined`. When this is the case, we need to call this block of code to set the `selectedTokenAddress`. 
`setSelectedTokenAddress(selectedSellToken.address);`

Reply:
fixed